### PR TITLE
Move animated-container to code excerpts and enable null safety

### PR DESCRIPTION
**Pull request by miquelbeltran**
Here I moved the example from the cookbook recipe animated-container.md to a project, used code excerpts to extract the code and enabled null safety in the dartpad.

cc. @sfshaza2 @RedBrogdon

### CODE REVIEW

File: src/docs/get-started/codelab.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ as well as many other open source packages, on [pub.dev][].
        flutter:
          sdk: flutter
        cupertino_icons: ^1.0.2
-    +  english_words: ^4.0.0
+    +  english_words: ^4.0.0-0
```
Comment (miquelbeltran) [inline src/docs/get-started/codelab.md:226]: For some reason this is changed by the `./tool/refresh-code-excerpts.sh` script, even if I didn't change the original file

Comment (sfshaza2) [reply src/docs/get-started/codelab.md:226]: Yes, this is from a PR that @domesticmouse created and I merged and it broke the build. I'm not quite sure why it broke the build, as 4.0.0 *is* the latest release for english_words. But, in any case, it's nothing to do with your PR.  :)

**Comment by sfshaza2** [comment]
Please review, @domesticmouse!

File: src/docs/get-started/codelab.md (hunk 1 above)
Comment (domesticmouse) [reply src/docs/get-started/codelab.md:226]: The issue is that I didn't use `./tool/refresh-code-excerpts.sh` to update this file. I just updated it directly.

Please the original file to have `english_words: ^4.0.0` as the dep constraint

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**